x-pack/plugins/stack_connectors/server/connector_types/security/tines/index.ts

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import {
+  SubActionConnectorType,
+  ValidatorType,
+} from '@kbn/actions-plugin/server/sub_action_framework/types';
+import { SecurityConnectorFeatureId } from '@kbn/actions-plugin/common';
+import { urlAllowListValidator } from '@kbn/actions-plugin/server';
+import {
+  TINES_CONNECTOR_ID,
+  TINES_TITLE,
+} from '../../../../common/connector_types/security/tines/constants';
+import {
+  TinesConfigSchema,
+  TinesSecretsSchema,
+} from '../../../../common/connector_types/security/tines/schema';
+import { TinesConfig, TinesSecrets } from '../../../../common/connector_types/security/tines/types';
+import { TinesConnector } from './tines';
+import { renderParameterTemplates } from './render';
+
+export const getTinesConnectorType = (): SubActionConnectorType<TinesConfig, TinesSecrets> => ({
+  id: TINES_CONNECTOR_ID,
+  name: TINES_TITLE,
+  Service: TinesConnector,
+  schema: {
+    config: TinesConfigSchema,
+    secrets: TinesSecretsSchema,
+  },
+  validators: [{ type: ValidatorType.CONFIG, validator: urlAllowListValidator('url') }],
+  supportedFeatureIds: [SecurityConnectorFeatureId],
```

Comment:
@ymao1 Should not it show "Security solution"? 

<img width="1448" alt="Screenshot 2022-11-08 at 7 52 21 PM" src="https://user-images.githubusercontent.com/7871006/200642221-7f50fb73-b5bc-4168-a04f-73bd3921a003.png">

Reply:
No, we are showing the feature ID category now, which is either `Alerting Rules` or `Cases`. Since this is available for Security rules, that falls under `Alerting Rules`

Reply:
Oh, I was not aware of that. Thanks for clarifying!